package.json

```diff
@@ -191,6 +191,13 @@
                                 "type": "command",
                                 "command": "workbench.action.terminal.focus"
                             },
+                            {
+                                "key": "$",
```

Comment:
why the `$` key? 🤔

Reply:
@MarcoIeni  Just a thought, wondering if grouping all of these into a prefix would make more sense now that we have quite a bit of command related to terminal.

- `SPC ! !`  -> Show terminal
- `SPC ! "`  -> Open new external terminal
- `SPC ! '`  -> Show terminal
- `SPC ! r'`  -> Run recent command

Reply:
I think many people might have `SPC !` in their muscle memory to open the terminal. This is also the approach from spacemacs, right?
So I wouldn't break it :/
btw, now that I think about it, the `$` reminds me of the terminal prompt, so it's not a bad key :)

Reply:
`SPC !` prompts for running shell command in spacemacs, we remapping to focus on terminal. `$` reminds me of shell but not necessarily rerunning the recent command. However, I am open to either way.

Reply:
> SPC ! prompts for running shell command in spacemacs

I see, then I prefer to keep `SPC !` to do something, instead of opening a submenu.
I think `SPC ! !` might be too long since users are already used to `SPC !`. :/

Reply:
Let's try it. We can always update this :)